client/app/scripts/liveblog-edit/module.js

```diff
@@ -56,6 +56,7 @@ define([
 
         // define the $scope
         angular.extend($scope, {
+            publishDisabled: true,
```

Comment:
True it is missing.
It is used to disable or enable the publish button, and it was there in the view.
It seems that a merge has modify it, I will add it.